From 8df8f766e8196a1b921a337ba7e6e3943b3e9bc2 Mon Sep 17 00:00:00 2001
From: Tomasz Torcz <tomek@pipebreaker.pl>
Date: Wed, 9 Jul 2014 13:37:50 +0200
Subject: [PATCH] hostnamed: update documentation with new "watch" chassis type

(cherry picked from commit efab8d0b0ebf6d715949b7af66fecaf5e5e5d77b)
(cherry picked from commit 710348ce7349b224ba58bae765611455320f68d1)
---
 man/hostnamectl.xml  | 3 ++-
 man/machine-info.xml | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/man/hostnamectl.xml b/man/hostnamectl.xml
index a0776f55bc..f3038b3251 100644
--- a/man/hostnamectl.xml
+++ b/man/hostnamectl.xml
@@ -236,7 +236,8 @@
                                 <literal>laptop</literal>,
                                 <literal>server</literal>,
                                 <literal>tablet</literal>,
-                                <literal>handset</literal>, as well as
+				<literal>handset</literal>,
+				<literal>watch</literal>, as well as
                                 the special chassis types
                                 <literal>vm</literal> and
                                 <literal>container</literal> for
diff --git a/man/machine-info.xml b/man/machine-info.xml
index 7448e68fd4..244e9b69cf 100644
--- a/man/machine-info.xml
+++ b/man/machine-info.xml
@@ -138,7 +138,8 @@
                                 <literal>laptop</literal>,
                                 <literal>server</literal>,
                                 <literal>tablet</literal>,
-                                <literal>handset</literal>, as well as
+				<literal>handset</literal>,
+				<literal>watch</literal>, as well as
                                 the special chassis types
                                 <literal>vm</literal> and
                                 <literal>container</literal> for
